backend/inspirehep/matcher/config.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+GROBID_URL = "https://grobid.inspirebeta.net"
+GROBID_PROCESS_CITATION_URL = f"{GROBID_URL}/api/processCitation"
```

Comment:
If you do it this way, you can't override only the `GROBID_URL` through the environment/other config file.